### PR TITLE
Add Project Loved: Week of April 8th

### PR DESCRIPTION
**Pull request by cl8n**
waiting on forum links and then ready to go ⚡️

### CODE REVIEW

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/osu/seashore-on-the-moon.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Camellia feat. Nanahira - Seashore on the Moon](https://osu.ppy.sh/beatmapsets/685242) by [melloe](https://osu.ppy.sh/users/2367616)  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+[Seashore on the Moon](https://osu.ppy.sh/beatmapsets/685242) is a gorgeous technical map by [melloe](https://osu.ppy.sh/users/2367616) that perfectly represents the soothing, almost trance like Camellia track that backs it. The map uses clear visual concepts which are executed very well and play incredibly smoothly, which is remarkable considering the technical complexity of the rhythms used! If you've never seen this map and want to see a great play showcasing it, check out [this liveplay FC](https://www.youtube.com/watch?v=b47VtGy55uY) from [FunOrange](https://osu.ppy.sh/users/2051389), the notable Canadian mouse player.
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- b47VtGy55uY) from [FunOrange
+ b47VtGy55uY) by [FunOrange
```

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/taiko/grief-and-malice.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Helblinde - Grief & Malice](https://osu.ppy.sh/beatmapsets/439241) by [S a n d](https://osu.ppy.sh/users/676578)
+
+Created by S a n d and Backfire, Grief & Malice is a popular beatmap that some of you have already played in Taiko World Cup 2017. This awesome map uses mild SV, kiai fountains and many streams all set to a catchy tune! The map is challenging without being too hard, and is fun for most players who are seeking a great map to play anytime, in any occasion: multiplayer, singleplayer, tournaments, etc.
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: If it was collabed with Backfire, shouldn't his name also appear on the banner and as one of the authors above this line?

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/taiko/uber-6.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[PrimeMIX - Uber 6](https://osu.ppy.sh/beatmapsets/332051) by [snowball112](https://osu.ppy.sh/users/2350722), [Onicorn](https://osu.ppy.sh/users/3936770) and [Okoratu](https://osu.ppy.sh/users/1623405)
+
+Familiar to those who participated in the TWC : Last Man Standing 3 Tournament, this is a bit of a peculiar map, as it was in the Grand Finals Free Mod bracket, but also it seems that it's background was a real hit with the Japanese team that ended up winning, all of them having corresponding avatars (which can actually still be seen on IceOC's user page!). The mapset itself has a variety of difficulties for everyone to enjoy, and we hope everyone appreciates it in the same way that everyone appreciates the background!
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- TWC : Last
+ TWC: Last
```
? not sure if this is one of those things to ignore

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+
+[Chroma - sink to the deep sea world](https://osu.ppy.sh/beatmapsets/650973) by [\_Asriel](https://osu.ppy.sh/users/566276)  
+*written by [SartNYAN OwO xD](https://osu.ppy.sh/users/4100941)*
+
+sink to the deep sea world by [\_Asriel](https://osu.ppy.sh/users/566276) is a fairly new, but also very well acclaimed map. For this one you are definitely in need of fast fingers and great control over anti-flow type of patterns. [\_Asriel](https://osu.ppy.sh/users/566276) has become known for making some of the most challenging osu!catch maps in recent times, with some of his maps being quite clunky and questionable playability wise. This map of his, however, has been modded and alternated with multiple times and has become a very polished product of [\_Asriel](https://osu.ppy.sh/users/566276)'s. Anyhow, go have a shot at this one, break a leg, and then return over here to make your voting decision.
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/catch/apparition.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Nanahoshi Kangengakudan - IMAGINARY LIKE THE JUSTICE](https://osu.ppy.sh/beatmapsets/507020) by [-Ken](https://osu.ppy.sh/users/4430811)
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: This line and line 185 are reversed (or something more complicated than that happened).

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+
+[Nanahoshi Kangengakudan - IMAGINARY LIKE THE JUSTICE](https://osu.ppy.sh/beatmapsets/507020) by [-Ken](https://osu.ppy.sh/users/4430811)  
+*written by [SartNYAN OwO xD](https://osu.ppy.sh/users/4100941)*
+
+Apparition from [Razor Sharp](https://osu.ppy.sh/users/3414261) is definitely one hell of a map. This super strenuous, fast pased, hyper-heavy map is a very neat challenge for top of the line players. It is also not your typical song you would expect from osu!, it being technological death metal. Apparition has seen plenty of play in multiplayer lobbies in the past, showing just how well known this underappreciated map from [Razor Sharp](https://osu.ppy.sh/users/3414261) has become through the course of time. So, if you are up for the challenge and believe you can have a shot at this behemoth, go and vote!
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/catch/imaginary-like-the-justice.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Spawn Of Possession - Apparition](https://osu.ppy.sh/beatmapsets/425956) by [Razor Sharp](https://osu.ppy.sh/users/3414261)
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: This line and line 178 are reversed (or something more complicated than that happened).

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+We're kicking off the list with a high note this time around, with a chart by the hated and beloved, but utmostly respected for his charting expertise charter, [Shoegazer](https://osu.ppy.sh/users/2520707) and his rendition of [Fallujah's Sapphire](https://osu.ppy.sh/beatmapsets/452446). Everyone in the mania community knows Shoegazer for his breakcore charts and excellent charting quality which as one of his many staples are almost always criminally underrated. This chart is one of many of Shoegazer's charts featured in 4K Mania World Cup 2016, it was a semifinals freemod pick. It's 240 BPM jumpstream with hard patterning, but it rarely gets dense enough to affect most of the players' performance (combo-wise). Most of the jumpstream runs are quite spread out as well, though weaker players might find the jumpstream in the midsection to be quite draining due to the length of that section, it also has some tricky and sudden 1/3 rhythms that can trip players up if they don't pay enough attention. With 27 favourites and great community support, this is a must have in the Loved category!
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- pick. It's 240
+ pick. Its 240
```

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/mania/monkey-bussines.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[kors k - monkey bussines](https://osu.ppy.sh/beatmapsets/179709) by [cimpucimpu](https://osu.ppy.sh/users/2854623)  
+*written by [Pope Gadget](https://osu.ppy.sh/users/2288341)*
+
+[monkey bussines](https://osu.ppy.sh/beatmapsets/179709) is a joke that got way out of hand. But hey, it's still funny to troll lobbies with! Frankly it's a cute attempt at an SV chart and as such it's turned into the modern-day mania equivalent of a Rick Roll. As such there's something somewhat reminiscent about it, be it a call-back to everyone's first chart where rules were little established. What a joyous time it was. And yes I would find it bloody hilarious to see people battle it out for #1.
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- And yes I
+ And yes, I
```
:shrug:

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 8]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/mania/no-money-down-low-monthly-payments.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Nero's Day At Disneyland - No Money Down, Low Monthly Payments](https://osu.ppy.sh/beatmapsets/550208) by [Ciel](https://osu.ppy.sh/users/1455309)  
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+Now we're kicking off the second half of the loved list with a classic, [Ciel](https://osu.ppy.sh/users/1455309)'s SV behemoth, [No Money Down, Low Monthly Payments](https://osu.ppy.sh/beatmapsets/550208) is an SV chart that was created mainly for 2017's 7K Mania World Cup and it's an amazing piece of artwork which is fairly simple in structure but masterful in execution. It offers wide variety of SV gimmicks paired with unusual layering, tricky bursts and unpredictability of the song itself to create the experience which makes you want more. It's violent, it's unpredictable, but it's not a beast you can't tame. With enough practice, every decent 7K player can give this one a fair run for it's money.
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- for it's money.
+ for its money.
```

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 9]
```diff
@@ -0,0 +1,279 @@
...
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+Next up is a song that is a classic amongst older rhythm game players, a song from EZ2DJ and THE BMS OF FIGHTERS 2009 - revolutionary battles -, one of the most recognizable songs from those games, [Messier 333 by sun3](https://osu.ppy.sh/beatmapsets/89535) charted by an old Korean charter [MOONWOLF](https://osu.ppy.sh/users/228917) with guest difficulties by [Marirose](https://osu.ppy.sh/users/1899763) (formerly known as xeona), [Nijigen-kun](https://osu.ppy.sh/users/4635760) (listed here as 2D) and [leqek](https://osu.ppy.sh/users/1517607). This set features four keymodes: 4,5,6 & 7K and it has a full spread of difficulties mainly testing your typical BMS skillsets, streams, density and an insane amount of bracketing on 7K Extra difficulty. It will test your physical abilities as well as your mental stamina and general prowess at the keymode of your choice. Good luck and don't get lost in the rice!
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- keymodes: 4,5,6 & 7K and
+ keymodes: 4, 5, 6, & 7K and
```
not too sure :shrug:

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 10]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/mania/starry-presto.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Halozy - Starry Presto](https://osu.ppy.sh/beatmapsets/178027) by [Entozer](https://osu.ppy.sh/users/277044) and jnnno  
+*written by [aitor98](https://osu.ppy.sh/users/3154852)*
+
+More Long Notes! This time we have [Entozer]( https://osu.ppy.sh/users/277044) coming up with a very special map called [Starry Presto by Halozy](https://osu.ppy.sh/beatmapsets/178027). Featuring 2 different keymodes, this set mixes both Long Notes and Chordstreams making it really hard to hit and read properly. Talking a little bit about [Entozer]( https://osu.ppy.sh/users/277044), he has been around osu!mania since 2010, being able to rank one of the firsts non-converted maps in the history of this gamemode. He does have a few other remarkable charts such as [Music Revolver](https://osu.ppy.sh/beatmapsets/130464), [Little Star](https://osu.ppy.sh/beatmapsets/138430) and [polygon](https://osu.ppy.sh/beatmapsets/132000) which are definitely worth to give it a shot.
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- Featuring 2 different
+ Featuring two different
```
not sure

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 11]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/mania/monkey-bussines.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[kors k - monkey bussines](https://osu.ppy.sh/beatmapsets/179709) by [cimpucimpu](https://osu.ppy.sh/users/2854623)  
+*written by [Pope Gadget](https://osu.ppy.sh/users/2288341)*
+
+monkey bussines is a joke that got way out of hand. But hey, it's still funny to troll lobbies with! Frankly it's a cute attempt at an SV chart and as such it's turned into the modern-day mania equivalent of a Rick Roll. As such there's something somewhat reminiscent about it, be it a call-back to everyone's first chart where rules were little established. What a joyous time it was. And yes, I would find it bloody hilarious to see people battle it out for #1.
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- As such there's 
+ As such, there's 
```
I am fairly certain that there should be a comma there.

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 12]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/mania/mottai-night-land.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Kyary Pamyu Pamyu - Mottai Night Land](https://osu.ppy.sh/beatmapsets/309474) by [Cuppp](https://osu.ppy.sh/users/2345293)  
+*written by [aitor98](https://osu.ppy.sh/users/3154852)*
+
+A lot of people wanted to see this map loved, and here comes the chance for it to be! Kyary Pamyu Pamyu - Mottai Night Land is a known map in the 7K community, made by Cuppp in an extremely unfriendly way. The chart starts calmly with a few Long Notes every here and there, but once you are ten seconds in, it goes WILD. Really harsh Long Note releases, high density inverted Chordstream sections, shields and way more things that will make you crazy. Did I mention this map has Long Notes? I don't think so…
```
Comment (MegaApplePi) [inline news/2018-04-08-project-loved-week-of-april-8th.md]:
```diff
- I don't think so…
+ I don't think so...
```

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 13]
```diff
@@ -0,0 +1,279 @@
...
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+We're kicking off the list with a high note this time around, with a chart by the hated and beloved, but utmostly respected for his charting expertise charter, Shoegazer and his rendition of Fallujah's Sapphire. Everyone in the mania community knows Shoegazer for his breakcore charts and excellent charting quality which as one of his many staples are almost always criminally underrated. This chart is one of many of Shoegazer's charts featured in the 4K Mania World Cup 2016; it was a semifinals freemod pick. It has 240 BPM jumpstream with hard patterning, but it rarely gets dense enough to affect most of the players' performance (combo-wise). Most of the jumpstream runs are quite spread out as well, though weaker players might find the jumpstream in the midsection to be quite draining due to the length of that section, it also has some tricky and sudden 1/3 rhythms that can trip players up if they don't pay enough attention. With 27 favourites and great community support, this is a must have in the Loved category!
```
[the remark this answers is not in the document]
Comment (cl8n) [reply news/2018-04-08-project-loved-week-of-april-8th.md]: woops, "fixed" it incorrectly in my last commit. now actually fixed 👍

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 14]
```diff
@@ -0,0 +1,279 @@
+---
+layout: post
+title: "Project Loved: Week of April 8th"
+date: 2018-04-08 18:00:00 +0000
+---
+
+Another Sunday marks another [Project Loved](https://osu.ppy.sh/community/forums/120) voting on the horizon! Come and vote for your favourite maps nominated from all four game modes!
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: maps to beatmaps

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 15]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello and welcome to the first [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of April 2018! This week contains several exciting picks for every game mode, and has something for both new and old players!
+
+For those of you who are unfamiliar with Project Loved or how the new system works, each mode has "captains" who pick a selection of your submissions to be put up for vote every two weeks. These chosen selections are each given their own community poll, and any map with an approval rating above 75% after a week's time gets moved into the Loved category. Maps that are Loved have leaderboards, but don't award performance points (much like Qualified maps).
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: gets -> is
Maps -> Beatmaps
don't -> do not
much like -> very much like

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 16]
```diff
@@ -0,0 +1,279 @@
...
+
+If you'd like to vote for any of the maps in this week's selection, just click the map's header image and it'll lead you to the poll!
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: maps to beatmaps

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 17]
```diff
@@ -0,0 +1,279 @@
...
+
+If you'd like to help out and submit maps *you* love to be considered for the Loved category, please use this [submission form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)!
+
+Once again, [to the beat!](https://www.youtube.com/channel/UCIFsWU-grp6UgGgMm57FiGQ) has created a video for each game mode showcasing our map selections, so if you'd like to see the entries in action, be sure to check out the videos below!
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: beatmap

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 18]
```diff
@@ -0,0 +1,279 @@
...
+
+Let's get started!
+
+### Navigation
+
+[osu!](#osu)  
+[osu!taiko](#osutaiko)
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: Make it a list I think?

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 19]
```diff
@@ -0,0 +1,279 @@
...
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/osu/no-title.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Reol - No title](https://osu.ppy.sh/beatmapsets/260141) by [cRyo\[iceeicee\]](https://osu.ppy.sh/users/835998)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+How could anyone forget one of the most iconic maps of 2015? cRyo\[iceeicee\]'s No Title was among the first sets submitted for the now staple song within osu!  and still remains as one of the most popular among them. Boasting 440+ favorites, this map was featured prominently in many community tournament pools, was a popular request for streamers, and all-around maintained a steady difficulty that gained it popularity from both top players and newer players wanting to test their skills. This is one of those maps people will wonder why it wasn't nominated sooner.
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: beatmaps
sooner -> earlier?

File: news/2018-04-08-project-loved-week-of-april-8th.md [hunk 20]
```diff
@@ -0,0 +1,279 @@
...
+*written by [aitor98](https://osu.ppy.sh/users/3154852)*
+
+xi - over the top is a map created by ArcherLove that is mainly focused in 7K, with a wide range of difficulties that makes this set accessible for every kind of player, no matter how skilled you are. It does have a few Guest Difficulties as well, done by AncuL and Eternalie, which helped this set a lot in terms of pattern variety. The map has both Chordstream and Long Note based difficulties, so even if there is a difficulty that you might not like due to the patterns, pretty sure you'll find another one that will meet your standards.
+
+---
+
+From everyone on the Project Loved team, we hope you enjoy this week's selection, and look forward to the next! The winning maps will hit the [beatmap listing](https://osu.ppy.sh/beatmapsets?s=8) sometime next week and will be announced via the [osugame twitter](https://twitter.com/osugame).
+
+If you have any suggestions for maps to be added to the Loved category, please submit them using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)!
```
Comment (TPGPL) [inline news/2018-04-08-project-loved-week-of-april-8th.md]: beatmaps

File: news/2018-04-08-project-loved-week-of-april-8th.md (hunk 15 above)
Comment (cl8n) [reply news/2018-04-08-project-loved-week-of-april-8th.md]: why add "very"?

**Pull request merged**